flink-runtime/src/test/java/org/apache/flink/runtime/testutils/TestJvmProcess.java

```diff
@@ -260,6 +260,10 @@ public long getProcessId() {
 				Field pidField = clazz.getDeclaredField("pid");
 				pidField.setAccessible(true);
 				return pidField.getLong(process);
+			} else if (clazz.getName().equals("java.lang.ProcessImpl")) {
+				Method pid = clazz.getDeclaredMethod("pid");
+				pid.setAccessible(true);
```

Comment:
IS this necessary even though the method is public?

Reply:
@zentol , yes, if not setAccessible(true), the exception(`cannot access a member of class java.lang.ProcessImpl (in module java.base) with modifiers "public"`) will be thrown.